docs/TESTS.md

```diff
@@ -1,77 +1,42 @@
-Setup needed (only needs to be done once):
-
-```bash
-$ cd /path/to/exercism/lfe
-$ rebar get-deps compile
-$ export ERL_LIBS=$(find deps -maxdepth 1 -mindepth 1 | tr '\n' ':')
-```
-
-If you're not using Bash, change the last command to what is needed to set an
-environment variable in your shell, e.g. for [fish][2]:
-
-```fish
-$ set -xU ERL_LIBS (find deps -maxdepth 1 -mindepth 1 | tr '\n' ':')
-```
-
-[2]: http://fishshell.com
-
 For each example, the following general steps are required.
 
-First, compile the module and its tests.
+First, change directory to the exercise you want to practice, in
+this case `bob`.
 
 ```bash
-$ cd /path/to/exercism/lfe/<exercise>
+$ cd </path/to/exercism>/lfe/bob
 ```
 
-```bash
-$ mkdir -p ebin
-$ lfec -o ebin src/<module>.lfe test/<module>-tests.lfe
-```
+Next run the tests.  The `test` make target depends on `compile`,
+so running `make test` will ensure your source is compiled,
+ and then proceed to run the tests.
 
-or
 
 ```bash
-$ cd bob
-$ make compile
+$ make test
 ```
 
-Then run the tests.
+Congratulations! You have failing tests!!!
+
+Now read `README.md` to find out what you need to do to get all
+your tests passing.
 
-```bash
-$ lfe -pa ebin -noshell -eval \
-  "(eunit:test '<module>-tests '(verbose))" \
-  -s init stop
-```
 
-or
+Fix the first error reported by the test, and run the tests again.
 
 ```bash
 $ make test
 ```
 
-NB The `test` make target depends on `compile`, so to compile
-and test, `make test` is all that's required.
+The `test` make target depends on `compile`, so running `make test`
+will ensure your source is compiled, and then proceed to run the tests.
 
+Continue by fixing the next failing test in your source code, and
+running your test until you get all the test passing.
 
-Here's a specific example, using the "bob" exercise:
-
-```bash
-$ cd bob
-$ mkdir -p ebin
-$ lfec -o ebin src/bob.lfe test/bob-tests.lfe
-$ lfe -pa ebin -noshell -eval \
-  "(eunit:test 'bob-tests '(verbose))" \
-  -s init stop
-```
-
-Or, using `make`:
-
-```bash
-$ make test
-```
+Once all your test pass, take a final look over your code in the `src`
+directory to ensure you are happy with your code and submit for review.
```

Comment:
Was trying to find the best place for submission, but didn't find a good place on the core site for Exercism.  Should we link to the `Try It!` on the main LFE page (http://exercism.io/languages/lfe)?

If so, would it make sense to have a sub-headers under `Try It!`? e.g. `'("Fetching Exercises", "Configure Exercism", "Submitting Exercises")`?